src/Common/DTOs/AddressTypes/Root.cs

```diff
@@ -0,0 +1,6 @@
+namespace SSW.Rewards.Shared.DTOs.AddressTypes;
+
+public class Root
+{
+    public List<Result> results { get; set; }
```

Comment:
Same annoying warning as above

Reply:
As per our conversation, we will address this separately. Task captured in #763.